code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -196,7 +196,8 @@ All foods are distributed among various categories. Use common sense.
 			return FALSE
 	if(user.a_intent != INTENT_DISARM)
```

Comment:
Side note: Why does this check to see if you're not on disarm intent instead of checking to see if you're not on harm intent?

Reply:
disarm has some other interactions reserved iirc